lib/run.js

```diff
@@ -82,6 +87,11 @@ function run (opts, cb) {
 
   function onError () {
     errors++
+    if (opts.bailout) bailout()
```

Comment:
you can avoid this if and move the bailour logic in here.
Set a default for bailout to the max integer.